ImperatorToCK3/Source/Imperator/Genes/AccessoryGeneTemplate.cpp

```diff
@@ -0,0 +1,34 @@
+#include "AccessoryGeneTemplate.h"
+#include "Log.h"
+#include "ParserHelpers.h"
+#include "WeightBloc.h"
+
+
+ImperatorWorld::AccessoryGeneTemplate::AccessoryGeneTemplate(std::istream& theStream)
+{
+	registerKeys();
+	parseStream(theStream);
+	clearRegisteredKeywords();
+}
+
+void ImperatorWorld::AccessoryGeneTemplate::registerKeys()
+{
+	registerKeyword("index", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleInt indexInt(theStream);
+		index = indexInt.getInt();
+	});
+	registerRegex("male|female|boy|girl", [this](const std::string& ageSexStr, std::istream& theStream) {
+		const auto sexAge = commonItems::singleItem(ageSexStr, theStream);
+		if (sexAge.find('{') != std::string::npos)
+		{
+			std::stringstream tempStream(sexAge);
+			auto ageSexBloc = std::make_shared<WeightBloc>(tempStream);
+			ageSexWeightBlocs.insert(std::pair(ageSexStr, ageSexBloc));
+		}
+		else // for copies: "boy = male"
+		{
+			ageSexWeightBlocs.insert(std::pair(ageSexStr, ageSexWeightBlocs.find(sexAge)->second));
```

Comment:
What if find() returns .end() due to a broken mod? Check to prevent crashes.

Reply:
✔️ Changed the else to `else if (ageSexWeightBlocs.find(sexAge) != ageSexWeightBlocs.end()) // for copies: "boy = male"`